src/hotspot/share/gc/z/zDriver.cpp

```diff
@@ -301,6 +315,14 @@ void ZDriver::concurrent_process_non_strong_references() {
   ZHeap::heap()->process_non_strong_references();
 }
 
+void ZDriver::pause_unload_class() {
```

Comment:
fininsh_non_strong_refs is not paused. Shall we rename it to "unload_class"?

Reply:
renamed